docs/Guides/Contributing.md

```diff
@@ -7,20 +7,21 @@ help us.
 
 > ## Note
 > This is an informal guide. Please review the formal
-> [CONTRIBUTING document](/CONTRIBUTING.md) for full details and our
+> [CONTRIBUTING document](https://github.com/fastify/fastify/blob/main/CONTRIBUTING.md) for full details and our
 > [Developer Certificate of Origin](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin).
 
 ## Table Of Contents
-<a id="contributing-toc"></a>
```

Comment:
Please do not change the attribute from `id` to `name`. `name` was deprecated from the `a` element many years ago.

Reply:
Sure, I'll switch them all from name to id then!